pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -341,8 +406,9 @@ public String getDataDir() {
   }
 
   public int getSegmentCommitTimeoutSeconds() {
```

Comment:
I feel that these method names also need to be changed, or have a new method here and mark the old one as deprecated. 
The behavior should be that the new one should be used if both are specified or only the new one be specified. If not, use the old one.

Reply:
"The behavior should be that the new one should be used if both are specified or only the new one be specified" - Isn't it what it is doing @jackjlli? Basically if the new config is absent, it falls-back to the older one. But method still does what it says - `getSegmentCommitTimeoutSeconds`.

Reply:
Right, I wrote this one because I saw some of the java docs for the methods have been adjusted but it talks about the other way around. So mention it here.